scripts/generate-ssl-cert.sh

```diff
@@ -71,6 +71,10 @@ mkdir -p $DIR/var/lib/jelastic/keys/
 rm -f $DIR/var/lib/jelastic/keys/*.pem
 
 [ ! -z $certdir ] && cp -f $certdir/* $DIR/var/lib/jelastic/keys/ && chown jelastic -R $DIR/var/lib/jelastic/keys/
+[ "$withExtIp" == "false" ] && {
+    cp -f "/var/lib/jelastic/SSL/jelastic.crt" "/var/lib/jelastic/SSL/jelastic.crt-backup";
```

Comment:
https://github.com/jelastic-jps/lets-encrypt/blob/master/scripts/auto-update-ssl-cert.sh#L55
it's better to copy all files to ${DIR}/var/lib/jelastic/keys/ without renaming  (and only in case ${DIR}/var/lib/jelastic/keys/ exists)